web/src/components/shared/Ads/useBoardFilter.ts

```diff
@@ -228,6 +227,19 @@ export const useBoardFilter = ({
     );
   }, [filter, filterParams, handleChangeFilter, history, lang, paymethods]);
 
+  useEffect(() => {
+    if (
+      paymethods &&
+      paymethods.length > 0 &&
+      !filterParams.paymethod &&
+      filterParams.paymethodSlug
+    ) {
+      handleChangeFilter({
+        paymethod: paymethods?.find((pm) => pm.slug === filterParams.paymethodSlug)?.id,
```

Comment:
```suggestion
        paymethod: paymethods.find((paymethod) => paymethod.slug === filterParams.paymethodSlug)?.id,
```
Рекомендую писать без сокращений